src/main/java/gregtech/integration/exnihilo/recipes/ExNihiloRecipes.java

```diff
@@ -0,0 +1,234 @@
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ModItems.pebbles, 4, 3))
+                .outputs(new ItemStack(Blocks.STONE, 1, 5))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 0))
+                .outputs(OreDictUnifier.get(stone, Basalt, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 1))
+                .outputs(OreDictUnifier.get(stone, GraniteBlack, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 2))
+                .outputs(OreDictUnifier.get(stone, Marble, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+        COMPRESSOR_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(ExNihiloModule.GTPebbles, 4, 3))
+                .outputs(OreDictUnifier.get(stone, GraniteRed, 1))
+                .EUt(4)
+                .duration(40)
+                .buildAndRegister();
+
+        // Crushed stones
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .input(Blocks.SAND)
+                .output(ModBlocks.dust)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(Blocks.STONE, 1, 1))
+                .output(ModBlocks.crushedGranite)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(Blocks.STONE, 1, 3))
+                .output(ModBlocks.crushedDiorite)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .inputs(new ItemStack(Blocks.STONE, 1, 5))
+                .output(ModBlocks.crushedAndesite)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .input(Blocks.NETHERRACK)
+                .output(ModBlocks.netherrackCrushed)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+        FORGE_HAMMER_RECIPES.recipeBuilder()
+                .input(Blocks.END_STONE)
+                .output(ModBlocks.endstoneCrushed)
+                .EUt(16)
+                .duration(10)
+                .buildAndRegister();
+
+        // Meshes
+        if (ExNihiloConfig.harderMeshes) {
```

Comment:
I feel like for the harder mesh recipes, the center item should be a ring, to hold everything together.